presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -393,7 +394,10 @@ protected void setup(Binder binder)
         binder.bind(FinalizerService.class).in(Scopes.SINGLETON);
 
         // Spiller
-        binder.bind(SpillerFactory.class).to(BinarySpillerFactory.class).in(Scopes.SINGLETON);
+        install(installModuleIf(
```

Comment:
Initially this was implemented as a plugin, but during our offline discussion (in April as far as I remember?) you or @martint specifically requested. If I remember correctly @martint explicitly referenced `NetworkTopology` :) Or do you have something else in mind?